StackViewController/StackViewContainer.swift

```diff
@@ -271,7 +271,7 @@ public class StackViewContainer: UIView, UIScrollViewDelegate {
         precondition(index < items.endIndex)
         
         let item = items[index]
-        if items.count >= 1 && index == items.endIndex.predecessor() {
+        if items.count >= 1 && index == items.endIndex.predecessor() && index > 0 {
```

Comment:
If index is 0 the precondition passes. No?

Jeff Falcon   Partner, CTO   917-523-3729
TANKA   Brooklyn, NY

> On Aug 20, 2016, at 6:42 PM, Indragie Karunaratne <notifications@github.com> wrote:
> 
> In StackViewController/StackViewContainer.swift:
> 
> > @@ -271,7 +271,7 @@ public class StackViewContainer: UIView, UIScrollViewDelegate {
> >          precondition(index < items.endIndex)
> >          
> >          let item = items[index]
> > -        if items.count >= 1 && index == items.endIndex.predecessor() {
> > +        if items.count >= 1 && index == items.endIndex.predecessor() && index > 0 {
> The additional index > 0 check here is unnecessary because of the precondition(index >= items.startIndex) line above.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>